understand_data.R

```diff
@@ -0,0 +1,19 @@
+# Glimpse of data and missing value check
+understand_data<- function(data) {
```

Comment:
nit: spaces on both sides of `<-`

You can automate finding/fixing the nit-picky stuff using the R packages described here: https://style.tidyverse.org/